doc/radosgw/qat-accel.rst

```diff
@@ -33,13 +33,20 @@ QAT Environment Setup
    encryption and compression services. And QAT driver in kernel space have to
    be loaded to drive the hardware.
 
-The driver package can be downloaded from `Intel Quickassist Technology`_.
+The out-of-tree QAT driver package can be downloaded from `Intel Quickassist
+Technology`_.
 
-2. The implementation for QAT based encryption is directly base on QAT API which
-   is included the driver package. But QAT support for compression depends on
-   QATzip project, which is a user space library which builds on top of the QAT
-   API. Currently, QATzip speeds up gzip compression and decompression at the
-   time of writing.
+The QATlib can be downloaded from `qatlib`_, which is used for the in-tree QAT
+driver.
+
+   .. note::
+      The out-of-tree QAT driver is gradually turning to intree driver+QATlib.
```

Comment:
What does “turning to” means?  I think we want something like “is being migrated to” instead.

Reply:
I agree.